tests/ea/mutation/mutator/test_mutate.py

```diff
@@ -49,7 +49,9 @@ def _test_mutate(mutator, record, mutation_record):
     assert (
         result.get_mutator_name() == mutation_record.get_mutator_name()
     )
+    original_mol = result.get_molecule_record().get_molecule()
+    mutated_mol = mutation_record.get_molecule_record().get_molecule()
     is_equivalent(
-        result.get_molecule_record().get_molecule(),
-        mutation_record.get_molecule_record().get_molecule(),
+        original_mol.with_canonical_atom_ordering(),
```

Comment:
If you move `with_canonical_atom_ordering` up to where the variable is decclared this function call can fit on one line. If it doesn't, keyword arguments need to be used on each line.

Reply:
yep